modulefiles/module_base.wcoss_dell_p3

```diff
@@ -61,3 +45,13 @@ module load python/3.6.3
 
 # waveprep
 module load cdo/1.9.8
+
+# Workaround for github hpc-stack issues #370
+setenv LMOD_EXACT_MATCH no
+module load netcdf/4.7.4
+module load nemsio/2.5.2
+module load w3emc/2.7.3
+module load upp/10.0.6
+module load wgrib2/2.0.8
+setenv WGRIB2 wgrib2
```

Comment:
Please move this back to where they were in the module load.

Reply:
These module need not to be load with LMOD_EXACT_MATCH=no. Therefore they should be load before the LMOD_EXACT_MATCH=no statement.

Reply:
Answered in Files changed page.